.pre-commit-config.yaml

```diff
@@ -47,6 +47,14 @@ repos:
         name: gen-docs
         entry: python hooks/gen_docs.py
         language: system
-        types: [python]
+        files: > # contains `.gitignore` as an indicator whether the script is ran with `--all-files`
+          (?x)^(
+            kpops/cli/main\.py
```

Comment:
shouldn't the docs be updated on any Python source file changing (except for tests), i.e. components?

Reply:
WDYT now?